docs/02-app/01-building-your-application/01-routing/06-redirecting.mdx

```diff
@@ -252,6 +252,78 @@ export const config = {
 
 See the [Middleware](/docs/app/building-your-application/routing/middleware) documentation for more information.
 
+## External redirects
+
+The App Router fully supports programmatic redirects to external links using the same methods mentioned above.
+
+[`redirect`](#redirect-function) and [permanentRedirect`](#permanentredirect-function) work on the server and in [Server Actions](/docs/app/building-your-application/data-fetching/server-actions-and-mutations).
```

Comment:
Can remove the second paragrah imo, since it's mentioned in the overview table. 

```suggestion
`redirect` and `permanentRedirect` can be used for external links using absolute URLs. 
```